testing/ef_tests/src/cases/kzg_recover_cells_and_kzg_proofs.rs

```diff
@@ -62,6 +62,7 @@ impl<E: EthSpec> Case for KZGRecoverCellsAndKZGProofs<E> {
                     .recover_cells_and_compute_kzg_proofs(
                         cell_indices.as_slice(),
                         input_cells.as_slice(),
+                        input_proofs.as_slice(),
```

Comment:
FyI the spec tests are currently passing even without this because we do a workaround to compare the output with the inputs, to satisfy the current spec tests - should be able to remove that altogether once the spec PR is merged.

https://github.com/sigp/lighthouse/pull/5998/files#diff-910a7a6a592415f79d81b45a2f8ab500b09ce31bb2e69c85af2bbb1c125a321dR73-R83

Reply:
Would need to wait for a spec release too I think?

An alternative is to tentatively modify the cryptography code and remove all of test vectors that specifically involve invalid proofs -- would you prefer this?

Reply:
I don't really know if we need these changes - If I understand your explanation correctly, the PR is to make the `recover_cells_and_compute_kzg_proofs` api match the current consensus spec _temporarily_ - given that the proofs parameter is likely going to be removed in the next spec release - and we don't actually use the proofs array at all. So it seems like we can avoid this change so we don't have to worry about removing it later, right? It doesn't actually change what we're testing in the spec tests.  Or am I missing something?

Reply:
That's correct -- the main issue is that the cryptography implementations are following the specs so they will fail if not given the proofs

Reply:
I see. The `proofs` is not required in the current version of c-kzg used on `das` branch. Is this change to prepare support for newer version of c-kzg / or the rust peerdasKzg library?

Reply:
That branch uses an older version of c-kzg, before this spec change was added

Reply:
It got removed yesterday from c-kzg following that spec test PR, so spec tests will be failing in c-kzg until we merge that consensus-spec PR: 

https://github.com/ethereum/c-kzg-4844/commit/9af09715877f74e6966210283153aa73bd1ba41f

I can do the same in peerdas-kzg